float64 timeStamp      # ROS time-stamp
int64[] forces        # Palm sensor values
